python/dgl/transforms/module.py

```diff
@@ -557,19 +557,19 @@ class AddSelfLoop(BaseTransform):
 
      >>> transform = AddSelfLoop(fill_data='sum')
      >>> g = dgl.heterograph({
-    ...     ('user', 'follows', 'user'): (torch.tensor([1, 2]),
-    ...                                   torch.tensor([0, 1])),
-    ...     ('user', 'plays', 'game'): (torch.tensor([0, 1]),
-    ...                                 torch.tensor([0, 1]))})
-    >>> g.edata['feat'] = {('user', 'follows', 'user'): torch.randn(2, 5),
-    ...                    ('user', 'plays', 'game'): torch.randn(2, 5)}
-    >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
-      ...                   ('user', 'plays', 'game'): torch.randn(2, 15)}
+     ...     ('user', 'follows', 'user'): (torch.tensor([1, 2]),
+     ...                                   torch.tensor([0, 1])),
+     ...     ('user', 'plays', 'game'): (torch.tensor([0, 1]),
+     ...                                 torch.tensor([0, 1]))})
+     >>> g.edata['feat'] = {('user', 'follows', 'user'): torch.randn(2, 5),
+     ...                    ('user', 'plays', 'game'): torch.randn(2, 5)}
+     >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
+     ...                    ('user', 'plays', 'game'): torch.randn(2, 15)}
```

Comment:
```suggestion
     >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
     ...                     ('user', 'plays', 'game'): torch.randn(2, 15)}
```

Reply:
Still fail after apply the suggested change.

Reply:
solve this in 5843